auditors/nonroot/fixtures/run-as-user-psc-0-allowed-multi-containers-single-label.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod
+  labels:
+    name: pod
+    container.audit.kubernetes.io/container1.allow-run-as-root: "SuperuserPrivilegesNeeded"
```

Comment:
I'm a bit confused why setting an override label for the container results in `runAsUser: 0` being allowed on the pod... Is this the behaviour we want? Is it consistent with `runAsNonRoot`?

Reply:
Hum indeed, the override label should be for the pod or container2 if the goal is to allow them to run as root. container1 won't run as root anyway as it runs with user 1.

I will check that test.